src/plugins/signoff/sagas.js

```diff
@@ -130,9 +130,12 @@ export function* handleApproveChanges(getState, action) {
   const {bucket} = getState();
   try {
     const collection = yield call(_updateCollectionAttributes, getState, {status: "to-sign"});
+
+    yield call(onCollectionRecordsRequest, getState, {bid: bucket.data.id, cid: collection.data.id});
```

Comment:
Maybe add a comment like *Simulate page refresh to catch new timestamps* or something like that